docs/src/main/tut/user-guide.md

```diff
@@ -441,7 +441,7 @@ For all `String`-based endpoints, Finch provides an `as[A]` method to perform ty
 is available for any `Endpoint[String]`, `Endpoint[Option[String]]` or `Endpoint[Seq[String]]` as
 long as a matching implicit `io.finch.DecodeEntity[A]` type-class is in the scope.
 
-This facility is designed to be intuitive, meaning that you do not have to provide a
+This facility is designed to be intuitive, meaning that you do not have to provide an
```

Comment:
@rpless Thanks for your review.
I thought there's one occurrence [here](https://finagle.github.io/finch/cookbook.html#fixing-the-toservice-compile-error) in the doc. But right now I noticed that it's an `Encoder` that follows the `an`.
I'm fine dropping `io.finch` or something else we agree.